tests/functional/test_home.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.home import HomePage
+
+
+@pytest.mark.nondestructive
+def test_promos_are_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.length == 16
+
+
+@pytest.mark.nondestructive
+def test_tweet_is_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.tweet_is_present
```

Comment:
I try to keep these sorts of additional to a minimum, but this will add value. We could also have an is_element_displayed when needed. It was always implicit waits that made these messy, and we're not using those. :smile: